CHANGELOG.md

```diff
@@ -1,3 +1,109 @@
+### citus v6.0.0 (November 4, 2016) ###
+
+* Adds compatibility with PostgreSQL 9.6, now the recommended version
+
+* Removes the `pg_worker_list.conf` file in favor of a `pg_dist_node` table
+
+* Removes the `\stage` command and corresponding csql binary in favor of `COPY`
+
+* Removes `copy_to_distributed_table` in favor of first-class `COPY` support
+
+* Adds support for multiple DDL statements within a transaction
+
+* Adds support for certain foreign key constraints
+
+* Adds limited support for `INSERT INTO ... SELECT FROM` commands
```

Comment:
Maybe note something like "parallel and distributed execution of insert/select on colocated tables" ? Just an attempt to highlight the feature a bit more.